cluster.go

```diff
@@ -189,6 +189,15 @@ func (c *Cluster) Validate(isTmpl bool) error {
 		}
 	}
 
+	nodeAddressSet := make(map[string]struct{})
+	for _, n := range c.Nodes {
+		_, ok := nodeAddressSet[n.Address]
+		if ok {
```

Comment:
This can be simplified:
```suggestion
		if _, ok := nodeAddressSet[n.Address]; ok {
```